### ISSUE TITLE
F&L Vet Center redirects

### ISSUE THREAD

**Issue by jilladams** [department-of-veterans-affairs/va.gov-cms#9640]
## Description

A set of requested redirects under the path `https://www.va.gov/directory/guide/` are blocked by information we need from the Facilities and Leadership Directory team (Kevin Reid). F&L has been updated with a feature to allow F&L editors to add URLs to redirect to Vet Center pages. Until the governance / process there is understood, we are not implementing these redirects. 

8/30 update: 
F&L redirect via #9547 is no-op. Let's implement following the existing paradigm: 

### For all `facility.asp` paths 
https://github.com/department-of-veterans-affairs/devops/blob/c449e4af26278eb175cf2ca1f48da390e759e073/ansible/deployment/config/revproxy-vagov/vars/redirects.yml#L1075

Original issue links for code comment:
```
https://github.com/department-of-veterans-affairs/va.gov-cms/issues/9515
https://github.com/department-of-veterans-affairs/va.gov-cms/issues/9218
https://github.com/department-of-veterans-affairs/va.gov-cms/issues/9813
https://github.com/department-of-veterans-affairs/va.gov-cms/issues/10373
```
Current URL  |  Redirect Destination or New URL
---  |  ---
https://www.va.gov/directory/guide/facility.asp?ID=5622&dnum=All | https://www.va.gov/macon-vet-center/
https://www.va.gov/directory/guide/facility.asp?ID=471&dnum=1 | https://www.va.gov/little-rock-vet-center/
| https://www.va.gov/directory/guide/facility.asp?ID=618&dnum=1 | https://www.va.gov/billings-vet-center/ |
| https://www.va.gov/directory/guide/facility.asp?ID=5462&dnum=1 | https://www.va.gov/fort-worth-vet-center/ |
| https://www.va.gov/directory/guide/facility.asp?ID=5464&dnum=1 | https://www.va.gov/mcallen-vet-center/ |
| https://www.va.gov/directory/guide/facility.asp?ID=5940&dnum=1 | https://www.va.gov/spring-vet-center/ |
https://www.va.gov/directory/guide/facility.asp?ID=652&dnum=All | https://www.va.gov/savannah-vet-center/
https://www.va.gov/directory/guide/facility.asp?ID=593&dnum=All | https://www.va.gov/scranton-vet-center/
https://www.va.gov/directory/guide/facility.asp?ID=6162&dnum=All | https://www.va.gov/columbus-ga-vet-center/
| https://www.va.gov/directory/guide/facility.asp?ID=5487&dnum=All | https://www.va.gov/harlem-vet-center/ |
https://www.va.gov/directory/guide/facility.asp?ID=6135&dnum=All | https://www.va.gov/clermont-vet-center/
https://www.va.gov/directory/guide/facility.asp?ID=6357&dnum=All | https://www.va.gov/augusta-vet-center/
https://www.va.gov/directory/guide/facility.asp?ID=6362&dnum=1 | https://www.va.gov/abilene-vet-center/
https://www.va.gov/directory/guide/facility.asp?ID=6132&dnum=All | https://www.va.gov/prince-georges-county-vet-center/
https://www.va.gov/directory/guide/facility.asp?ID=5443&dnum=All | https://www.va.gov/fort-myers-vet-center/
https://www.va.gov/directory/guide/facility.asp?ID=5456&dnum=1 | https://www.va.gov/new-orleans-vet-center/
https://www.va.gov/directory/guide/facility.asp?ID=5654&dnum=All | https://www.va.gov/middletown-vet-center/
https://www.va.gov/directory/guide/facility.asp?ID=411&dnum=1 | https://www.va.gov/charleston-sc-vet-center/

### For all `keystaff.asp` paths
https://github.com/department-of-veterans-affairs/devops/blob/c449e4af26278eb175cf2ca1f48da390e759e073/ansible/deployment/config/revproxy-vagov/vars/redirects.yml#L1664

Original issue links for code comment:
```
https://github.com/department-of-veterans-affairs/va.gov-cms/issues/9515
https://github.com/department-of-veterans-affairs/va.gov-cms/issues/9218
https://github.com/department-of-veterans-affairs/va.gov-cms/issues/9813
https://github.com/department-of-veterans-affairs/va.gov-cms/issues/10373
```
Current URL  |  Redirect Destination or New URL
---  |  ---
https://www.va.gov/directory/guide/keystaff.cfm?id=5622 | https://www.va.gov/macon-vet-center/
https://www.va.gov/directory/guide/keystaff.cfm?id=471 | https://www.va.gov/little-rock-vet-center/
| https://www.va.gov/directory/guide/keystaff.cfm?id=618 | https://www.va.gov/billings-vet-center/ |
| https://www.va.gov/directory/guide/keystaff.cfm?id=5462 | https://www.va.gov/fort-worth-vet-center/ |
| https://www.va.gov/directory/guide/keystaff.cfm?id=5487 | https://www.va.gov/harlem-vet-center/ |
| https://www.va.gov/directory/guide/keystaff.cfm?id=5464 | https://www.va.gov/mcallen-vet-center/ |
| https://www.va.gov/directory/guide/keystaff.cfm?id=5940 | https://www.va.gov/spring-vet-center/ |
https://www.va.gov/directory/guide/keystaff.cfm?id=652 | https://www.va.gov/savannah-vet-center/
https://www.va.gov/directory/guide/keystaff.cfm?id=593 | https://www.va.gov/scranton-vet-center/
https://www.va.gov/directory/guide/keystaff.cfm?id=6162 | https://www.va.gov/columbus-ga-vet-center/
https://www.va.gov/directory/guide/keystaff.cfm?id=6135 | https://www.va.gov/clermont-vet-center/
https://www.va.gov/directory/guide/keystaff.cfm?id=6357 | https://www.va.gov/augusta-vet-center/
https://www.va.gov/directory/guide/keystaff.cfm?id=6362 | https://www.va.gov/abilene-vet-center/
https://www.va.gov/directory/guide/keystaff.cfm?id=6132 | https://www.va.gov/prince-georges-county-vet-center/
https://www.va.gov/directory/guide/keystaff.cfm?id=5443 | https://www.va.gov/fort-myers-vet-center/
https://www.va.gov/directory/guide/keystaff.cfm?id=5456 | https://www.va.gov/new-orleans-vet-center/
https://www.va.gov/directory/guide/keystaff.cfm?id=619 | https://www.va.gov/fayetteville-ar-vet-center//
https://www.va.gov/directory/guide/keystaff.cfm?id=5706 | https://www.va.gov/fayetteville-ar-vet-center/
https://www.va.gov/directory/guide/keystaff.cfm?id=5654 | https://www.va.gov/middletown-vet-center/
https://www.va.gov/directory/guide/keystaff.cfm?id=411 | https://www.va.gov/charleston-sc-vet-center/

### Implementation date
When does this request need to be live: ASAP 

## Additional context
- Original requests: 
  - https://github.com/department-of-veterans-affairs/va.gov-cms/issues/9515
  - https://github.com/department-of-veterans-affairs/va.gov-cms/issues/9218
  - https://github.com/department-of-veterans-affairs/va.gov-cms/issues/9813
- Discovery tickets: 
  - https://github.com/department-of-veterans-affairs/va.gov-cms/issues/9175
  - https://github.com/department-of-veterans-affairs/va.gov-cms/issues/9546

## Acceptance Criteria

- [ ] After deploy, check original URLs on production to confirm that each properly redirects and resolves to the requested URL before closing ticket

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide Crew`
- [ ] `⭐️ Sitewide CMS`
- [X] `⭐️ Public Websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`

**Comment by jilladams** [comment]
Also: with any follow up code changes we need to make sure to add the Github issue URLs in the ticket body here into the code comments on these 2 sections of the redirect yml, por favor, e.g. https://github.com/department-of-veterans-affairs/devops/blob/master/ansible/deployment/config/revproxy-vagov/vars/redirects.yml#L1073

**Comment by jtmst** [comment]
@jilladams   These redirects do seem to be working, with the exception of id=619 because of the collision noted above.  How should that be handled?

**Comment by jtmst** [comment]
Closed, but follow up required to double check ID=619

**Comment by jilladams** [comment]
619 follow up: https://dsva.slack.com/archives/C01K37HRUAH/p1663690304076669
Will file a new ticket if SWC determines we have more to do for that.
